src/WIP-Draft - Governance Process.md

```diff
@@ -38,9 +38,13 @@ providers and consumers - in direct connection with one another, reducing interm
 all users to act in the marketplace knowing that they have *supreme transparency and fair price
 discovery*.
 
-We are determined to make WindingTree Protocol the **canonical marketplace** for the trading of 
+We are determined to make WindingTree Protocol the **recognized marketplace** for the trading of 
 *real-world services* by applying continuous innovation, encouraging intellectual discourse and
-expanding the WindingTree Protocol ecosystem - inviting the brightest minds to join the community. 
+expanding the WindingTree Protocol ecosystem - inviting the brightest minds to join the community.
+
+Some examples of marketplaces that can be transformed with the WindingTree protocol are Online Travel Agencies
```

Comment:
I think that we should be a little more general and not so prescriptive, otherwise we limit our scope and flexibility. How about something like: "WindingTree seeks to disrupt established real-world service marketplaces across industries including, but not limited to, travel, ticketing, and the sharing economy."